### DISCUSSION TITLE
failing to load the dashboard in dev environment

### DISCUSSION THREAD

**Discussion by hyouinKyouma** [apache/superset#14386] [Q&A / Help]
when in explorer view, interaction with the data is possible, moving it to dashboard throws error. I have attached the snapshot. This happens only in dev environment. 

### Expected results

should be able to navigate to the dashboard.

#### Screenshots

![errorDashboard](https://user-images.githubusercontent.com/37413995/115965976-0c26d800-a549-11eb-8caf-fe4f83bbe090.png)
![errorDashboard2](https://user-images.githubusercontent.com/37413995/115965978-0e893200-a549-11eb-96bd-1cde16fc31f8.png)
![errorDashboard3](https://user-images.githubusercontent.com/37413995/115965980-121cb900-a549-11eb-9ef8-cff7e312a5b7.png)

### Environment

(please complete the following information):

- superset version: `1.1.0`
- python version: `3.8.5`
- node.js version: `10.19.0`,
- OS: `ubuntu20.04`

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by junlincc** [comment]
@hyouinKyouma did it happen recently? or it's first time setting up?
